interface/src/app/node/node.js

```diff
@@ -304,6 +307,32 @@ angular.module('adage.node', [
         $scope.numExpShown =
           $scope.topMode ? $scope.topNum : $scope.experiments.length;
       };
+
+      // Event handler when user clicks the checkbox in front of each sample.
+      // It keeps track of the number of selected samples in each experiment
+      // to indicate whether the "show annotations" button should be enabled.
+      $scope.toggleSelection = function(exp, selected) {
+        if (selected) {
+          ++exp.numSelections;
+        } else {
+          --exp.numSelections;
+        }
+      };
```

Comment:
No, it would just make the code's intent easier to understand when someone else is reading it. I don't have strong feelings on it though, I am good with this implementation if you are good with it.